src/app/shared/services/mock.services.ts

```diff
@@ -0,0 +1,143 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class MockService {
+
+  constructor() { }
+
+  getHomepageData(): Observable<any> {
+    const data = {
+      data: {
+        content: {
+          id: '1',
+          entityRevisionId: '22',
+          nid: 1,
+          title: 'Homepage',
+          moderationState: 'published',
+          status: true,
+          paragraphs: [
```

Comment:
isn't this a bit too much..? Why do we want to use the homepage as a showcase of all the variants for the banner, for instance? Bu also why having an accordion with all the items expanded, good that we have started populating an home page, but let's not pollute it.

Reply:
indeed, cleaned it but i still kept this data in a different method, in the case we will do a showcase page.